### DISCUSSION TITLE
How to change the default icon of .msi file ?

### DISCUSSION THREAD

**Discussion by JeremyLeyvraz** [oleg-shilo/wixsharp#1507] [Q&A] [answered]
Hello,
I managed to change all the icons, but I can't seem to change the icon of the generated .msi file.
Could you help me please ?
Thank you :)

**Comment by oleg-shilo** [comment, marked as answer]
I do not think you can change the icon of an individual msi file. You can only do this for all msi files on the system.

**Comment by JeremyLeyvraz** [comment]
Thanks for your answer !
